website/static/js/registrationUtils.js

```diff
@@ -291,16 +291,22 @@ var Question = function(questionSchema, data) {
  *
  * @param {function}: save: save function for the current registrationDraft
  **/
-Question.prototype.addComment = function(save) {
+Question.prototype.addComment = function(save, page, event) {
     var self = this;
 
     var comment = new Comment({
         value: self.nextComment()
     });
     comment.seenBy.push($osf.currentUser().id);
-    self.comments.push(comment);
-    self.nextComment('');
-    save();
+
+    var $comments = $(event.target).closest('.registration-editor-comments');
+    $osf.block('Saving...', $comments);
+    return save()
+        .always($osf.unblock.bind(null, $comments))
+        .done(function () {
+            self.comments.push(comment);
```

Comment:
> Error handling for the comment UI could get awkward-- type a comment, see it get added, then a red alert appears and the comment is deleted? Or moved back to a pending state?

I'd think it would be moved back to a pending state. Not that it matters so much. The overwhelmingly common case will be that the request succeeds, so better to optimize the UI for that case.

All that said, this PR is an improvement to the status quo, so I'll stop picking nits. =)